src/plugins/lanplus/lanplus.c

```diff
@@ -1449,34 +1431,32 @@ void getIpmiPayloadWireRep(
 
 	/* second checksum */
 	tmp = len - cs;
-	msg[len++] = ipmi_csum(msg+cs, tmp);
+	msg[len++] = ipmi_csum(msg + cs, tmp);
 
 	/* Dual bridged request: 2nd checksum */
 	if (bridgedRequest == 2) {
 		tmp = len - cs3;
-		msg[len++] = ipmi_csum(msg+cs3, tmp);
+		msg[len++] = ipmi_csum(msg + cs3, tmp);
 		payload->payload_length += 1;
 	}
 
 	/* bridged request: 2nd checksum */
 	if (bridgedRequest) {
 		tmp = len - cs2;
-		msg[len++] = ipmi_csum(msg+cs2, tmp);
+		msg[len++] = ipmi_csum(msg + cs2, tmp);
 		payload->payload_length += 1;
 	}
 }
 
 
-
 /*
  * getSolPayloadWireRep
  *
  * param msg [out] will contain our wire representation
  * param payload [in] holds the v2 payload with our SOL data
  */
-void getSolPayloadWireRep(
-						  uint8_t          * msg,     /* output */
-						  struct ipmi_v2_payload * payload) /* input */
+void getSolPayloadWireRep(uint8_t *msg,			   /* output */
```

Comment:
I don't think clang-format is happy with parameter comments, because it didn't seem to clean them all consistently.

Reply:
Maybe clang-format is just the wrong tool for this project? Don't you think so?